quadcopter_brain/src/quadcopter_brain/landing_site.py

```diff
@@ -47,11 +46,13 @@ def find_fiducial_center(self, x_coords, y_coords, z_coords, ids):
         Finds the center of the fiducial, in meters, from the camera
         TODO: Incorporate which fiducials are seen to find the center
         '''
+        assert(len(x_coords) == len(y_coords))
```

Comment:
again, this seems a bit unnecessary. it's not an exposed method (afaik), which means we should be passing in correctly formatted arguments